lib/product.service.ts

```diff
@@ -4,14 +4,20 @@ import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
 import { Construct } from 'constructs';
 import * as path from 'path';
 import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as AWS from 'aws-sdk';
+import { v4 as uuidv4 } from 'uuid';
+import { products } from './models/product.config';
+import { stock } from './models/stock.config';
+import * as dynamo from "aws-cdk-lib/aws-dynamodb";
 
 export class ProductStack extends Stack {
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
 
+    // Serverless lambdas
     const lambdaProductsList = new NodejsFunction(this, 'lambda-product-list', {
```

Comment:
why do we have two versions? here and then a new one with dynamo db? lets update the existing one with new one? Same for "lambdaProductsById"